source/user-manual/ruleset/ruleset-xml-syntax/rules.rst

```diff
@@ -247,11 +247,11 @@ action
 
 Any action (decoded as the ACTION).
 
-+--------------------+------------------------------------------------------------------+
-| **Default Value**  | n/a                                                              |
-+--------------------+------------------------------------------------------------------+
-| **Allowed values** | Any `sregex expression <regex.html#os-match-or-sregex-syntax>`_  |
-+--------------------+------------------------------------------------------------------+
++--------------------+----------------------+
+| **Default Value**  | n/a                  |
++--------------------+----------------------+
+| **Allowed values** | Deny | Drop | Accept |
```

Comment:
I don't think that's correct.

`<action>` will match any normal string. I have successfully matched `<action>` against other text like `<action>denied</action>`

(Aside: it would be very helpful if action could match an sregex, but that would be a behaviour change.  Here we should just be documenting current behaviour)